test/unit/spec/EnrollCall_spec.js

```diff
@@ -292,7 +292,8 @@ function (Q, Okta, OktaAuth, LoginUtil, Util, Form, Beacon, Expect, $sandbox,
           expect(test.afterRenderHandler).toHaveBeenCalledTimes(1);
           expect(test.afterRenderHandler.calls.allArgs()[0]).toEqual([
             {
-              controller: 'enroll-call'
+              controller: 'enroll-call',
+              settings: jasmine.any(Object)
```

Comment:
Smell.
This should be checked with a custom matcher or at least a helper checker.

Reply:
Notice that with the used Jasmine version the *any* matcher fail to support *Settings* constructor.